oozie-to-airflow/templates/dag.tpl

```diff
@@ -1,6 +1,6 @@
 with models.DAG(
     '{{ dag_name }}',
     schedule_interval=datetime.timedelta(days=1),
-    start_date=datetime.datetime(2018, 11, 26)
+    start_date=airflow.utils.dates.days_ago(1)
```

Comment:
Instead of hardcoding, can this be configurable and instead default to 1? Same with timedelta above.